### PR TITLE
[v3.1.2] Release update

### PR DESCRIPTION
**Pull request by ogeorget-psee**

### CODE REVIEW

**Review by lbristiel-psee — APPROVED** [review]

File: cmake/custom_functions/add_library_version_header.cmake [hunk 1]
```diff
@@ -8,8 +8,8 @@
 # See the License for the specific language governing permissions and limitations under the License.
 
 set(GIT_BRANCH "HEAD")
-set(GIT_COMMIT_ID "37aa7510f3d6bdbd2b2b66e816d4fb0e3a420e9d")
-set(GIT_COMMIT_DATE "2022-11-23 15:21:14 +0100")
+set(GIT_COMMIT_ID "6a744effa905d697563baa52a50715ae69d8ba35")
```
Comment (nmartin-psee) [inline cmake/custom_functions/add_library_version_header.cmake:11]: I did not follow this change, but why do we need that ?

**Comment by nmartin-psee** [comment]
This is one single squashed commit ? 
In the past, we've been doing merge commit on `main` because rebasing on this branch is too much of a hassle in case of a major release, but we may still need to keep the commits history

**Review by nmartin-psee — CHANGES_REQUESTED** [review]

**Comment by nmartin-psee** [comment]
You're talking about OpenEB, I'm talking about Metavision ;)

File: cmake/custom_functions/add_library_version_header.cmake (hunk 1 above)
Comment (nmartin-psee) [reply cmake/custom_functions/add_library_version_header.cmake:11]: I am dumb ^^

**Review by nmartin-psee — APPROVED** [review]

**Pull request merged**